### PR TITLE
feat(): add next to asciidoc

### PR DESCRIPTION
**Pull request by isaac-shek**
#### 📲 What

A description of the change.

#### 🤔 Why
		
Why it's needed, background context.
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR.
		 
#### 🕵️ How to test

Notes on how a reviewer can test the changes, e.g. how to run the tests.

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [ ] Passing all automated tests, including a successful deployment?
- [ ] Passing any exploratory testing?
- [ ] Rebased/merged with latest changes from development and re-tested?
- [ ] Meeting the Coding Standards?

### CODE REVIEW

File: asciidoc/getting_started/next/plugin-information.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+nx list @ensono-stacks/next
+
+View additional information about a plugin capability through the following command:
+
+[source, bash]
+nx g @ensono-stacks/next:[generator-executor-name] --help
+
+== Generators
+
+<!-- markdownlint-disable MD033 -->
```
Comment (MarkComerford) [inline asciidoc/getting_started/next/plugin-information.adoc:48]: Need to remove this linting tag

File: asciidoc/getting_started/next/init.adoc [hunk 2]
```diff
@@ -0,0 +1,65 @@
+== @ensono-stacks/next:init
```
Comment (MarkComerford) [inline asciidoc/getting_started/next/init.adoc:1]: Could make the child pages collapsible. I've done this for cypress and playwright just to make it like for like with the current implementation.

File: asciidoc/getting_started/next/init.adoc [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+UPDATE apps/baseline-next-app/project.json #Updated with custom test config to allow for coverage collection
+UPDATE apps/baseline-next-app/.eslintrc.json #Ehanced with additional linting rules
+UPDATE apps/baseline-next-app/tsconfig.json #Minor enhancements
+UPDATE apps/baseline-next-app/tsconfig.spec.json #Updates for monorepo structure
+UPDATE apps/baseline-next-app/specs/index.spec.tsx #Formatting changes
+----
+
+The generator will also add https://github.com/dequelabs/axe-core-npm/blob/develop/packages/react/README.md[react-axe] (version 4.7.3) into the app via the following:
+
+....app.tsx
```
Comment (MarkComerford) [inline asciidoc/getting_started/next/init.adoc:46]: could change this to just .app.tsx so it acts as the header for the codeblock. Currently seems a bit out of place when viewed in the browser.

File: asciidoc/getting_started/next/next-auth-redis-deployment.adoc [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            {
+              "command": "helm upgrade [... unchanged ...] --set nextAuthSecret=\\\"$NEXTAUTH_SECRET\\\"",
+              "forwardAllArgs": false
+            }
+          ]
+        }
+      }
+----
+
+WARNING: For Azure DevOps, the **NEXTAUTH_SECRET** needs to be added to the <company\>-<component\>-<domain\>-nonprod and <company\>-<component\>-<domain\>-prod variable groups
```
Comment (leesaxby) [inline asciidoc/getting_started/next/next-auth-redis-deployment.adoc:116]: Is this how we expect the `<component>` bit here to render?
image

File: asciidoc/getting_started/next/next-auth-redis.adoc [hunk 5]
```diff
@@ -0,0 +1,98 @@
+== @ensono-stacks/next:next-auth-redis
+
+Add Redis for session management to your Next application
+
+The next-auth-redis generator will add Redis for session management into your existing Next app with Next-auth.
+
+=== Prerequisites
+
+An existing [Next](https://nextjs.org/) application with Next-auth. Use the `@ensono-stacks/next:next-auth` generator to add this into your application
```
Comment (leesaxby) [inline asciidoc/getting_started/next/next-auth-redis.adoc:9]: `[Next](https://nextjs.org/)` needs to be `https://nextjs.org/[Next]`

File: asciidoc/getting_started/next/plugin-information.adoc (hunk 1 above)
Comment (leesaxby) [inline asciidoc/getting_started/next/plugin-information.adoc:48]: Think we need to remove this `<!-- markdownlint-disable MD033 -->` as its being rendered on the page.

File: asciidoc/getting_started/next/next-auth-redis-deployment.adoc (hunk 4 above)
Comment (isaac-shek) [reply asciidoc/getting_started/next/next-auth-redis-deployment.adoc:116]: good question, the document renders like this - https://stacks.ensono.com/docs/getting_started/next/next-auth-redis-deployment

Comment (isaac-shek) [reply asciidoc/getting_started/next/next-auth-redis-deployment.adoc:116]: I've added a code quotes to be more visible

**Review by leesaxby — APPROVED** [review]

**Pull request merged**